enterprise_subsidy/apps/api/v1/serializers.py

```diff
@@ -51,7 +51,8 @@ class Meta:
             "internal_only",
             "revenue_category",
             "is_active",
-            "total_deposits"
+            "total_deposits",
+            "created",
```

Comment:
[consider] it might also be worth exposing `modified` date in this serializer, too? 🤔